.gitignore

```diff
@@ -117,3 +117,6 @@ schema-*.json
 
 # Infracost
 *.infracost
+
+# Azurite
```

Comment:
what's Azurite mean?

Reply:
Azurite is a dev tool for azure storage account emulation